lib/parsers/srtEntries.js

```diff
@@ -5,10 +5,20 @@ function parseSrtEntries(subtitle, options = {
   invalidEntries: true,
   invalidTimecodes: true,
 }) {
-  // Remove carriage returns
-  const newlined = subtitle.replace(/\r/g, '');
+  // Line breaks are OS dependent
+  // MAC  - \r
+  // UNIX - \n
+  // WIN  - \r\n
+  // Standardize files by replacing all line breaks with \n
 
-  // Remove newlines between lines
+  // Turn all \r to \r\n
+  // Note: \r\n will turn into \r\n\n - extra \n to be removed later
+  const carriaged = subtitle.replace(/\r/g, '\r\n');
+
+  // Remove all \r to leave just \n
+  const newlined = carriaged.replace(/\r/g, '');
```

Comment:
If this works great, but couldn't you just do `.replace(/\r\n/g, '')` to leave just \n?

Reply:
You bring up a great point, there's definitely a better way to do this.  The idea is to only have \n, because I parse based on that later.  The simplest way would be `.replace(/\r/g, '\n')` to just replace all instances of \r.  I'll make the change